Script/material.dat

```diff
@@ -1910,30 +1910,35 @@ flesh
     AttachedGod = MORTIFER;
     CategoryFlags = Base|IS_GOLEM_MATERIAL;
     InteractionFlags = Base|CAN_BURN;
+    BodyFlags = Base|IS_WARM_BLOODED;
```

Comment:
You should consider making `WARM_BLOODED` part of `Base` and then doing `Base|~IS_WARM_BLOODED` on the other cases.